.github/workflows/test.yml

```diff
@@ -30,6 +30,7 @@ jobs:
       - uses: actions/checkout@v2
       - run: |
           # Wait for startup of openssh-server
+          timeout 15 ./wait_for_sshd_start_up.sh
           sleep 15
```

Comment:
Did you mean to remove the `sleep 15` here too?

Reply:
Oops, good catch!

Reply:
Done!